javascript/packages/cli/src/cli.ts

```diff
@@ -126,7 +126,7 @@ program
     "<binaries...>",
     `the binaries that you want to be downloaded, provided in a row without any separators;\nThey are downloaded in current directory and appropriate executable permissions are assigned.\nPossible options: 'polkadot', 'polkadot-parachain'\n${decorators.blue(
       "zombienet setup polkadot polkadot-parachain",
-    )}`,
+    )}\nNote: Downloading 'polkadot' downloads also 'polkadot-prepare-worker' and 'polkadot-execute-worker'`,
```

Comment:
package polkadot into npm. inventing own package manager is dead end.

Reply:
@dzmitry-lahoda can you elaborate a bit more on this - what exactly you mean?

Reply:
Maybe he means that `zombienet setup` is doing a job that can already be done by a package manager?

Reply:
If that is the case then i think it is not correct; Cause not everyone is using zombienet by cloning the repo; There are cases of using it as a release - and thus the release can help setting up the binaries;

Reply:
yes. just ask user to use apt npm nix curl dpkg etc. 

adhoc pm is dead end. let start that it is not secure nor it will work on proxyvpnedfirewalled envs.

parity to release apt or and npm. 

parity already releases nix polkadot. waiting for nix common good nodes;

Reply:
just ask user to unstall, like you ask to install k8s, podman, etc.

on windows can use scoop or winget. 

on mac brew or nix.

Reply:
This was created as an easy way for non-advanced linux users to crash-test zombienet; It retrieves these from the release packages of gitlab and its not the "mandatory way to go";

I am fine for removing it if @pepoviola agrees - I do not see any value though on removing this;